sample-portfolio.ini

```diff
@@ -0,0 +1,14 @@
+[AAPL]
+graph=True
+buy=10@100
+buy=5@120
+sell=3@122
+sell=8@125
+color=#FFFF00 
+
+[SPKE]
+graph=False
+buy=1@10.28
+buy=1@10.30
+sell=1@10.32
+color=lime
```

Comment:
I don't think we need a gen sample config function. I've removed it and replaced it with a static sample file

Reply:
We had it in there just in case someone wanted to start fresh from the base config file. @lemonase originally implemented this feature, but I'm not opposed to removing it as the base file will always be in the repo.

Unless @lemonase has strong reservations about keeping it in I'm fine with the removal.

Reply:
cool - yeah I'd advocate for this. if/when we make changes to the config layout we can update here rather than the generator function